### DISCUSSION TITLE
Ignore classes, but still counting credits

### DISCUSSION THREAD

**Discussion by meadoge** [khongsomeo/GPA-OOP#79] [Ideas]
Some classes is required but its result only counted as `passed` or `failed`. I'm talking to classes like English, PE, ..etc.

Maybe it's time to build a new option?

**Comment by trhgquan** [comment]
Agreed. I'm creating a new issue for this.

**Comment by hwangswan** [comment]
I don't think this is a good idea, personally. The program itself allows us to ignore courses, and "ignoring" means we will not mentioning it anymore.

Counting credits of an ignored course seems weird.

**Reply by trhgquan** [reply]
Agreed too, now I'm realised that this is too "ambiguous" to implement. Closing related issue #80.
